src/stream/src/executor/mview/materialize.rs

```diff
@@ -191,6 +209,11 @@ impl<S: StateStore, SD: ValueRowSerde> MaterializeExecutor<S, SD> {
                 }
                 Message::Barrier(b) => {
                     self.state_table.commit(b.epoch).await?;
```

Comment:
After https://github.com/risingwavelabs/risingwave/pull/14681 we can call `commit_with_enable_consistent_op` when we set the can_disable_overwrite_conflict_check.